Modules/Setup

```diff
@@ -359,7 +359,7 @@ xxsubtype xxsubtype.c
 # Uncommenting the following line tells makesetup that all following modules
 # are not built (see above for more detail).
 #
-#*disabled*
+*disabled*
```

Comment:
Hummm.... this looks incorrect. Why is this being changed?

Reply:
My apologies, it appears that I left it there by mistake